common/components/UI/RealEstateComponents/RealEstateModal/index.tsx

```diff
@@ -97,6 +97,7 @@ const RealEstateModal: FC<Props> = ({
       onCancel={closeModal}
       okText={currentRealEstate ? 'Зберегти' : 'Додати'}
       cancelText={'Відміна'}
+      okButtonProps={{ style: { display: !editable ? 'none' : 'inline' } }}
```

Comment:
To hide button better use `okText=''` (empy `string` will hide the button)

Reply:
if I use the okText button with an empty string, then the close button shifts to the left
![image](https://github.com/user-attachments/assets/515dbc61-f593-4c2a-b127-e42c07ddf27e)
But if i use  okButtonProps={{ style: { display: !editable ? 'none' : 'inline' } }} button in needed position
![image](https://github.com/user-attachments/assets/c957d94b-807d-4ab2-aa80-dddae01ab0e3)

Reply:
Ok, agreed. Also some new info, the `okText=''` method isn't working on antd version 5.x, so...
But, let's make it look like this `{ ...(!editable && { display: 'none' }) }` to not override default antd button display style by using conditional object field